File: FrontendTemplateProject-main/src/app/models/Calendar-Tim/event.ts
import { Guid } from "guid-typescript";
import { EventType } from "./eventType";
import { Calendar } from "./calendar";

export class Event {
  eventId!: Guid;
  eventName!: String;
  eventDate!: Date;
  eventTime!: Date;
  eventDescription!: String;
  eventType!: EventType;
  calendar!: Calendar;
}

File: Grupa11-Calendar/Grupa11-Calendar/Models/Event.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Grupa11_Calendar.Models
{
    public class Event
    {
        public Guid EventId { get; set; }
        #region Event 
        public string EventName { get; set; }
        public DateTime EventDate { get; set; }
        public DateTime EventTime { get; set; }
        public string EventDescription { get; set; }
        public Guid EventTypeId { get; set; }
        [ForeignKey("EventTypeId")]
        public EventType EventType { get; set; }
        public Guid CalendarId { get; set; }
        [ForeignKey("CalendarId")]
        public Calendar Calendar { get; set; }  
        #endregion
    }
}
